File: PastaPHP/Model/exames.php
<?php 

require_once __DIR__ .'/../dao/ExamesDao.php';
//require_once __DIR__.'../Model/Exame.php'; 


class Exames {
    private $id;
    private $idUsuario; #chave extrangeira
    private $idPaciente; #chave extrangeira
    private $nomeExame;
    private $descricao;
    private $tipoAmostra;
    private $resultado;
    private $prioridade;

    public function __construct(){
        
    }

    /*public function __construct($id, $idUsuario, $idPaciente, $nomeExame, $descricao, $tipoAmostra, $resultado, $prioridade){
        $this->id = $id;
        $this->idUsuario = $idUsuario;
        $this->idPaciente = $idPaciente;
        $this->nomeExame = $nomeExame;
        $this->descricao = $descricao;
        $this->tipoAmostra = $tipoAmostra;
        $this->resultado = $resultado;
        $this->prioridade = $prioridade;

    }*/

    public function getId(){
        return $this->id;
    }
    
    public function setId($id){
        $this->id = $id; 
    }

    public function getIdUsuario(){
        return $this->idUsuario;
    }
    
    public function setIdUsuario($idUsuario){
        $this->idUsuario = $idUsuario; 
    }

    public function getIdPaciente(){
        return $this->idPaciente;
    }
    
    public function setIdPaciente($idPaciente){
        $this->idPaciente = $idPaciente; 
    }
    
     public function getNomeExame(){
        return $this->nomeExame;
    }
    
    public function setNomeExame($nomeExame){
        $this->nomeExame = $nomeExame; 
    }

    public function getDescricao(){
        return $this->descricao;
    }
    
    public function setDescricao($descricao){
        $this->descricao = $descricao; 
    }

    public function getTipoAmostra(){
        return $this->tipoAmostra;
    }
    
    public function setTipoAmostra($tipoAmostra){
        $this->tipoAmostra = $tipoAmostra; 
    }

    public function getResultado(){
        return $this->resultado;
    }
    
    public function setResultado($resultado){
        $this->resultado = $resultado; 
    }

     public function getPrioridade(){
        return $this->prioridade;
    }
    
    public function setPrioridade($prioridade){
        $this->prioridade = $prioridade; 
    }

    
        public function __toString(){
             return "Id Exame: {$this->id} - Id Usuário: {$this->idUsuario} - Id Paciente: {$this->idPaciente} - Nome do Exame: {$this->nomeExame} - Descrição: {$this->descricao} - Tipo da Amostra: {$this->tipoAmostra} - Resultado: {$this->resultado} - Prioridade: {$this->prioridade}";
        }

 }
?>

File: PastaJava/Model/Exames.java
package PastaJava.model;

public class Exames {
    private int id;
    private int idUsuario;
    private int idPaciente; 
    private String nomeExame;
    private String descricao;
    private String tipoAmostra;
    private String resultado;
    private String prioridade;

    public Exames (){}

    public Exames(int id, int idUsuario, int idPaciente, String nomeExame, String descricao, String tipoAmostra, String resultado, String prioridade) {
        this.id = id;
        this.idUsuario = idUsuario;
        this.idPaciente = idPaciente;
        this.nomeExame = nomeExame;
        this.descricao = descricao;
        this.tipoAmostra = tipoAmostra;
        this.resultado = resultado;
        this.prioridade = prioridade;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getIdUsuario() {
        return idUsuario;
    }

    public void setIdUsuario(int idUsuario) {
        this.idUsuario = idUsuario;
    }

    public int getIdPaciente() {
        return idPaciente;
    }

    public void setIdPaciente(int idPaciente) {
        this.idPaciente = idPaciente;
    }

    public String getNomeExame() {
        return nomeExame;
    }

    public void setNomeExame(String nomeExame) {
        this.nomeExame = nomeExame;
    }

    public String getDescricao() {
        return descricao;
    }

    public void setDescricao(String descricao) {
        this.descricao = descricao;
    }

    public String getTipoAmostra() {
        return tipoAmostra;
    }

    public void setTipoAmostra(String tipoAmostra) {
        this.tipoAmostra = tipoAmostra;
    }

    public String getResultado() {
        return resultado;
    }

    public void setResultado(String resultado) {
        this.resultado = resultado;
    }

    public String getPrioridade() {
        return prioridade;
    }

    public void setPrioridade(String prioridade) {
        this.prioridade = prioridade;
    }

    @Override
    public String toString() {
        return "Exames [id=" + id + ", idUsuario=" + idUsuario + ", idPaciente=" + idPaciente + ", nomeExame=" + nomeExame + ", descricao=" + descricao + ", tipoAmostra=" + tipoAmostra + ", resultado=" + resultado + ", prioridade=" + prioridade + "]";
    }

}
